README.adoc

```diff
@@ -1,16 +1,15 @@
 = J-Voting
-:groupId: io.github.oliviercailloux.y2018
+:groupId: io.github.j-voting
```

Comment:
Ok to remove y2018, but please keep io.github.oliviercailloux. The project will continue and I want it to stay under a gId that I control.

Reply:
@oliviercailloux Oh right, but yet you indicated to change the groupid, I used the one belonging to us, the one of the organization. Perhaps you should consider clarifying this point for next years.

Reply:
Good remark, indeed that was not clear.